### DISCUSSION TITLE
Wanderer's License

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8255] [Q&A] [answered]
Can someone explain how to get the various Wanderer's Licenses? The wiki doesn't provide much information apart from "You get it when you do this mission."

**Comment by warp-core** [comment, marked as answer]
Doing their missions should get you there. It's a fairly linear path.
If you are playing on continuous, you need to complete the Hai Reveal story and then land on th Wanderer capital, Vara K'chrai, and then the missions begin.
If you are on 0.9.16.1 or earlier, you need to have met the Hai and then land on Vara K'chrai to start the Wanderer missions.

In either case, after you complete the initial first contact stuff and are given access to their outfits, land on Vara Ke'sok and check the spaceport until you get a mission to defend the planet (it may take multiple attempts, the mission has a 50% chance to offer on each landing), this leads to a short series of missions that grant access to some Wanderer ships.
To continue from this point, you must have completed the Free Worlds story. If that's done, you should get the next mission from Vara K'chrai.

**Reply by MasterOfGrey** [reply]
> It's a fairly linear path.

Depressingly so...
